packages/alfa-rules/src/sia-r54/rule.ts

```diff
@@ -0,0 +1,54 @@
+import { Rule, Diagnostic } from "@siteimprove/alfa-act";
+import { Element } from "@siteimprove/alfa-dom";
+import { Predicate } from "@siteimprove/alfa-predicate";
+import { Err, Ok } from "@siteimprove/alfa-result";
+import { Page } from "@siteimprove/alfa-web";
+import { isIgnored } from "../common/predicate/is-ignored";
+
+import { expectation } from "../common/expectation";
+import { Node } from "@siteimprove/alfa-aria";
+
+const { and, not } = Predicate;
+const { isElement } = Element;
+
+export default Rule.Atomic.of<Page, Element>({
+  uri: "https://alfa.siteimprove.com/rules/sia-r54",
+  evaluate({ device, document }) {
+    return {
+      applicability() {
+        return document
+          .descendants({ flattened: true, nested: true })
+          .filter(isElement)
+          .filter(
+            and(not(isIgnored(device)), (element) =>
+              Node.from(element, device)
+                .attribute("aria-live")
+                .some((attribute) => attribute.value === "assertive")
```

Comment:
🤔 Thinking that we can probably add a `hasAriaAttribute(predicate: Predicate<aria.Attribute>): Predicate<Element>` in our predicates. That would be a generalisation of `hasHeadingLevel` that could also be used here and in a couple other rules.

But that's maybe topic for another PR…